database_scheme.sql

```diff
@@ -0,0 +1,45 @@
+CREATE TABLE "users" (
+  "id" integer PRIMARY KEY NOT NULL,
+  "first_name" varchar NOT NULL,
+  "second_name" varchar NOT NULL,
+  "password_hash" integer NOT NULL,
+  "city" varchar,
+  "birthday" date,
+  "avatar" varchar
+);
+
+CREATE TABLE "days" (
+  "id" integer PRIMARY KEY NOT NULL,
+  "user_id" integer REFERENCES users NOT NULL,
+  "date" date NOT NULL,
+  "activities_color" varchar,
+  "emotions_color" varchar
```

Comment:
What do these fields represent? If it is the information about the average colors for a day, then the database is oversaturated with information (because you can calculate their values using data from other parts of the database) and therefore the fields should be removed

Reply:
You got the meanings right, I thought they will represent average colors but I agree that we can calculate them using other data. I'll remove these fields